src/main/java/org/cactoos/text/EndsWith.java

```diff
@@ -43,6 +43,15 @@
      */
     private final Text suffix;
 
+    /**
+     * Ctor.
+     * @param origin The origin
+     * @param suffix The suffix
+     */
+    public EndsWith(final String origin, final String suffix) {
```

Comment:
@andreoss this is also out of scope of this PR.

Reply:
Added it for a test, which got removed. Done